src/App.js

```diff
@@ -1,16 +1,32 @@
 import React from 'react';
+const todoList = [
+  {
+    id:1,
+    title: "Complete assigment"
+  },
+  {
+    id:2,
+    title: "Run assigment"
+  },
+  {
+    id:3,
+    title: "Check assigment"
+  }
+];
 
 function App() {
   return (
-    <div style={{ textAlign: 'center' }}>
-      <header>
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a href="https://reactjs.org" target="_blank" rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+    <div>
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map(function(item) {
+          return <li key={item.id}> 
+          <span>{item.title}</span> 
+                </li>
+        }
+        )}
+
```

Comment:
```suggestion
        {todoList.map(function(item) {
          return <li key={item.id}> 
            <span>{item.title}</span> 
          </li>
        })}
```

Reply:
Nice! Each time you have a line break that goes further inside a function, or nests another component under the current level, try to indent so the structure of the code is a little more clear. 

This doesn't affect the execution of the code, which is already great, but does make the code more readable to you and other humans =D